composer.json

```diff
@@ -1,43 +1,41 @@
 {
-	"name": "silverstripe/dynamodb",
-	"description": "SilverStripe DynamoDB integration.",
-	"type": "silverstripe-vendormodule",
-	"keywords": ["silverstripe", "aws", "dynamodb"],
-	"license": "BSD-3-Clause",
-	"authors": [
-		{
-			"name": "SilverStripe",
-			"homepage": "http://silverstripe.com"
-		},
-		{
-			"name": "The SilverStripe Community",
-			"homepage": "http://silverstripe.org"
-		}
-	],
-	"require": {
-		"aws/aws-sdk-php": "~3.105",
-		"doctrine/cache": "1.*",
-		"silverstripe/framework": "^4@dev"
-	},
-	"require-dev": {
-		"phpunit/PHPUnit": "^5.7",
-		"squizlabs/php_codesniffer": "^3.0"
-	},
-	"autoload": {
-    "psr-4": {
-      "SilverStripe\\DynamoDb\\": "code/",
-      "SilverStripe\\DynamoDb\\Tests\\": "tests/"
-    }
-  },
-	"extra": {
-		"installer-name": "silverstripe-dynamodb",
-		"branch-alias": {
-			"dev-master": "4.0.x-dev"
-		}
```

Comment:
Removed branch alias as a matter of course.
Removed `installer-name` because it wasn't actually working anyway and is non-standard to ss modules.